packages/server/modules/workspaces/tests/integration/invites.graph.spec.ts

```diff
@@ -392,55 +442,88 @@ describe('Workspaces Invites GQL', () => {
         expect(sendEmailInvocations.args).to.have.lengthOf(count)
       })
 
-      itEach(
-        [InviteByTarget.Email, InviteByTarget.Id],
-        (type) => `works when inviting user by ${type}`,
-        async (type) => {
-          const sendEmailInvocations = EmailSendingServiceMock.hijackFunction(
-            'sendEmail',
-            async () => true
-          )
+      it('works when inviting user by id', async () => {
```

Comment:
not sure if I agree with splitting these out into 2, a lot of the test itself is still very similar